docs/section10/10-13B.md

```diff
@@ -0,0 +1,38 @@
+# Airside systems – Rule 10-13B      
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False  
+**Rule ID:** 10-13B      
+ 
+**Rule Description:** For proposed HVAC systems designed, mechanical preheating equipment efficiencies shall be adjusted to remove the supply fan energy from the efficiency rating.  
+
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Table G3.1 Section 10 b.  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. HVAC with mechanical heating.  
+
+**Function Call:** None  
+
+**Manual Check:**  
+
+ 1. Check that the mechanical equipment heating efficiency has been adjusted to remove the supply fan energy from the efficiency rating.   
+
+## Rule Logic:   
+**Applicability Check 1:**   
+- For each hvac system in the P_RMR: `for hvac_p in P_RMR...HeatingVentilationAirConditioningSystem:`
+    - For each preeating system in the HVAC system: `for preheating_system_p in hvac_p.preheat_system:`
+        - Check that heating system type is equal to heat_pump:: `if preheating_system_p.heating_system_type == "HEAT_PUMP":`
```

Comment:
similar comment as #371 and #375 - need a case 3 with the not applicable outcome if preheating doesn't exist or if heating type is not a heat pump